REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
I agree with not looking back. The management of two branches is error-prone and tedious IMO. 

I've had Compat in this package before and have to admit I'm not a fan of extra package requirements and vulnerability to bugs in that package. It would be one thing if this package had many more features planned but I don't see this as the case. 

And correct on `Pkg.update` managing the proper version for TimeSeries based on REQUIRE.

 If a package has TimeSeries as a dependency and supports Julia 0.4, the code block in that package's REQUIRE would read `TimeSeries 0.8.8 0.9.0`. Correct me if I'm wrong about this though as I've never used it that way.